pkgs/applications/audio/sptlrx/default.nix

```diff
@@ -15,10 +15,13 @@ buildGoModule rec {
 
   ldflags = [ "-s" "-w" ];
 
-  passthru.tests.version = testers.testVersion {
-    package = sptlrx;
-    # TODO Wrong version in `0.2.0`. Has been fixed upstream.
-    version = "v0.1.0";
+  passthru = {
+    updateScript = gitUpdater { inherit pname version; };
```

Comment:
I actually don't think `gitUpdater` can handle `vendorSha256` update, maybe you will want `nix-update-script` :thinking:

Reply:
Oh, good point. I'll have a look how other GoModules handle this.

Reply:
I think you are right, `nix-update-script` [should work]( https://github.com/NixOS/nixpkgs/blob/58303b3ef544dd04a16ea9579eb60626d121c339/pkgs/tools/virtualization/rootlesskit/default.nix#L17).

Reply:
Just tested it by plugging in the old `version`, `sha256` and `vendorSha256`. Worked just fine. Thx :)